qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -339,7 +344,7 @@ def _transpiled_circuits(self) -> List[QuantumCircuit]:
         """Return a list of experiment circuits, transpiled."""
         has_custom_transpile_option = (
             not set(vars(self.transpile_options)).issubset({"basis_gates", "optimization_level"})
-            or self.transpile_options.get("optimization_level", 0) != 0
+            or self.transpile_options.get("optimization_level", 1) != 1
```

Comment:
I don't see why, if the `optimization_level != default` you define this as custom transpilation, which means you eventually call `super()._transpiled_circuits()` and don't use the regular algorithm.

Reply:
This is intentional. See the comment above.